<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButton_start">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_stop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_serial">
         <property name="font">
          <font>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_upload">
         <property name="text">
          <string>Upload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_update">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_shutdown">
         <property name="text">
          <string>Shutdown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_msg">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>442</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_txt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pic">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_insoletype">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_batt">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QProgressBar" name="progressBar_1">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QProgressBar" name="progressBar_2">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QProgressBar" name="progressBar_3">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>4</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QProgressBar" name="progressBar_4">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QProgressBar" name="progressBar_5">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>6</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QProgressBar" name="progressBar_6">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>7</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QProgressBar" name="progressBar_7">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>8</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QProgressBar" name="progressBar_8">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>9</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QProgressBar" name="progressBar_9">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>10</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QProgressBar" name="progressBar_10">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>11</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QProgressBar" name="progressBar_11">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>12</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QProgressBar" name="progressBar_12">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>13</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QProgressBar" name="progressBar_13">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>14</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>15</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>16</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QProgressBar" name="progressBar_14">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QProgressBar" name="progressBar_15">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QProgressBar" name="progressBar_16">
           <property name="maximum">
            <number>1023</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
